rust/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "objecthash"
+version = "0.1.0"
+authors = ["Tiziano Santoro <tiziano88@gmail.com>"]
+
+[dependencies]
+rust-crypto = "^0.2"
```

Comment:
Ugh please no, it's completely unmaintained :cry:

Use this instead: https://github.com/RustCrypto/hashes